src/utilities/frame/types.ts

```diff
@@ -32,6 +32,8 @@ export interface ContextualSaveBarProps {
   discardAction?: ContextualSaveBarCombinedActionProps;
   /** Remove the normal max-width on the contextual save bar */
   fullWidth?: boolean;
+  /** Accepts a component that is ideally used to help users switch between different contexts */
```

Comment:
I would change this to be `Accepts a component that is used to help users switch between different contexts` to be a bit more opinionated about it.